packages/zowe-explorer/src/shared/context.ts

```diff
@@ -65,6 +65,7 @@ export function isJob(node: TreeItem): boolean {
  * @return true if a favorite pds, ds, false otherwise
  */
 export function isFavoritePsDs(node: TreeItem): boolean {
+    // TODO Fix for binary ds favorites
```

Comment:
I noticed this TODO here, I wasn't sure if this is planned for the draft PR or for the future, but figured I'd comment on it for awareness 😅